### PR TITLE
Update the Docs with Service Now Versions

### PR DESCRIPTION
**Pull request by dishanktiwari2501**
Signed-off-by: dishanktiwari2501 <dtiwari@progress.com>

### :nut_and_bolt: Description: What code changed, and why?

- Specify the ServiceNow versions on the following pages:
    - [Integration App](https://docs.chef.io/automate/servicenow_integration/)
    - [Incident App](https://docs.chef.io/automate/servicenow_incident_creation/)
- Add a note to the [Data Feeds](https://docs.chef.io/automate/datafeed/) page for the ServiceNow reference.
- Add a ServiceNow Notification section to the [Notification](https://docs.chef.io/automate/notifications/) page.

### :chains: Related Resources

Story: https://chefio.atlassian.net/browse/STALWART-329

### :athletic_shoe: How to Build and Test the Change

- Refer to the [Netlify](https://deploy-preview-7694--chef-automate.netlify.app/automate/servicenow_integration/) Link.

### :white_check_mark: Checklist

**All PRs** must tick these:

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [x] All commits signed off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [x] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [x] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [x] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [x] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [x] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [x] Tests added/updated? (all new code needs new tests)
- [x] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

File: components/docs-chef-io/content/automate/servicenow_incident_creation.md [hunk 1]
```diff
@@ -32,9 +32,13 @@ The Incident App generates a data stream of compliance events that you can lever
...
 
-## System Requirements
+### System Requirements
 
-* The [Integration App]({{< relref "servicenow_integration" >}}) is already installed and configured.
+* A running [Chef Automate](https://www.chef.io/automate/) instance.
+* Chef Automate has a valid SSL/TLS certificate from a trusted certificate authority (CA).
+* A running [ServiceNow](https://www.servicenow.com/) instance.
+* The ServiceNow instance is reachable on port 443.
+* Compatible with **Tokyo**, **San Diego** and **Rome** versions.
```
Comment (kalroy) [inline components/docs-chef-io/content/automate/servicenow_incident_creation.md:41]: This does not make a lot of sense to me

File: components/docs-chef-io/content/automate/servicenow_integration.md [hunk 2]
```diff
@@ -33,6 +33,7 @@ The Integration App works by exposing the REST API endpoints for communication b
 - Chef Automate has a valid SSL/TLS certificate from a trusted certificate authority (CA).
 - A running [ServiceNow](https://www.servicenow.com/) instance.
```
Comment (kalroy) [inline components/docs-chef-io/content/automate/servicenow_integration.md:34]:
```suggestion
- A running [ServiceNow](https://www.servicenow.com/) instance.* A running [ServiceNow](https://www.servicenow.com/) instance. The supported ServiceNow versions are **Tokyo**, **San Diego** and **Rome**.
```

File: components/docs-chef-io/content/automate/servicenow_integration.md [hunk 3]
```diff
@@ -33,6 +33,7 @@ The Integration App works by exposing the REST API endpoints for communication b
...
 - The ServiceNow instance is reachable on port 443.
+- Compatible with **Tokyo**, **San Diego** and **Rome** versions.
```
Comment (kalroy) [inline components/docs-chef-io/content/automate/servicenow_integration.md:36]: Does not make much sense to me.

File: components/docs-chef-io/content/automate/notifications.md [hunk 4]
```diff
@@ -13,23 +13,65 @@ gh_repo = "automate"
     weight = 10
 +++
 
-## About Notifications
+## Notifications
 
-Chef Automate notifications uses webhooks to send alerts for failures of Chef Infra Client runs and InSpec compliance scans. You can send notifications to Slack channels, Slack users, or any other service that supports webhook integrations. Notifications are sent for all failures. To ignore a failure, set the `ignore_failure` attribute to `true` on the resource. 
+Chef Automate notifications uses webhooks to send alerts for failures of Chef Infra Client runs and InSpec compliance scans. You can send notifications to Slack channels, Slack users, or any other service that supports webhook integrations. Notifications are sent for all failures. To ignore a failure, set the `ignore_failure` attribute to `true` on the resource.
```
Comment (kalroy) [inline components/docs-chef-io/content/automate/notifications.md:18]:
```suggestion
Chef Automate notifications uses webhooks to send alerts for failures of Chef Infra Client runs and InSpec compliance scans. You can send notifications to ServiceNow, Slack channels, Slack users, or any other service that supports webhook integrations. Notifications are sent for all failures. To ignore a failure, set the `ignore_failure` attribute to `true` on the resource.
```

File: components/docs-chef-io/content/automate/notifications.md [hunk 5]
```diff
@@ -13,23 +13,65 @@ gh_repo = "automate"
...
+To add a servicenow notification for Chef Automate, follow the steps given below:
+
+![ServiceNow notifications](/images/automate/servicenow-notifications-navigation.png)
+
+1. In the **Settings** tab, navigate to the _Notifications_ page in the sidebar.
+1. Select **Create Notification**.
+1. Select **ServiceNow**.
+1. Enter a unique notification name.
+1. Select the failure type to be notified on from the drop-down menu. Current options are Chef Infra Client run or InSpec scan.
+1. Get your ServiceNow webhook address by using the **What's this?** link, which opens an external servicenow site.
```
Comment (kalroy) [inline components/docs-chef-io/content/automate/notifications.md:47]: Please check if this is available or not.

**Review by kalroy — CHANGES_REQUESTED** [review]

File: components/docs-chef-io/content/automate/notifications.md (hunk 5 above)
Comment (dishanktiwari2501) [reply components/docs-chef-io/content/automate/notifications.md]: Checked and updated accordingly.

File: components/docs-chef-io/content/automate/notifications.md [hunk 6]
```diff
@@ -13,34 +13,79 @@ gh_repo = "automate"
...
-![Notifications Navigation](/images/automate/notifications-navigation.png)
+![Slack Navigation](/images/automate/notifications-navigation.png)
 
 1. In the **Settings** tab, navigate to the _Notifications_ page in the sidebar.
 1. Select **Create Notification**.
 1. Select **Slack**.
 1. Enter a unique notification name.
 1. Select the failure type to be notified on from the drop-down menu. Current options are Chef Infra Client run or InSpec scan
-1. Get your Slack webhook address by using the **What's this?** link, which opens an external Slack site.
+1. Get your Slack webhook address, which opens an external Slack site.
```
Comment (vivekshankar1) [inline components/docs-chef-io/content/automate/notifications.md:84]:
```suggestion
1. Get your Slack webhook address from [here](https://api.slack.com/messaging/webhooks).
```

**Review by vivekshankar1 — APPROVED** [review]

**Review by kalroy — APPROVED** [review]

**Pull request merged**